docs/src/explanations/instchoice.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title:  "Instance Choices"
+section: "chisel3"
+---
```

Comment:
```suggestion
```
We need to clean up the other files, but this header stuff was used by Jekyll, it is no longer used by the new Docusaurus website.